conf/modules.config

```diff
@@ -18,12 +18,57 @@ process {
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
-    withName: 'MULTIQC' {
+    withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:MULTIQC' {
```

Comment:
any reason to use the full name?

Reply:
It's just that we done that for all process.
This is not necessary, just for standardization.